test/genius_lyrics_test.dart

```diff
@@ -118,4 +118,12 @@ void main() {
     expect(tracks.any((element) => element.artist == 'J. Cole'), equals(true));
     expect(tracks.any((element) => element.lyrics!.isNotEmpty), equals(true));
   });
+
+  test('Testing searching song by lyrics snippet',
```

Comment:
i don't is a good idea testing like this , today the only result is Kendrick Song , but tomorrow could be some other song , maybe include or any it's better than equals...

Reply:
but if you change [this](https://github.com/hbtalha/genius_lyrics/pull/20#discussion_r1379129259), it can be possible